### ISSUE TITLE
Sunset Outdated Pages

### ISSUE THREAD

**Issue by exactlyallan** [rapidsai/rapids.ai#265]
As part of the larger site refresh, I think the below pages are now redundant and can instead be linked out. Suggestions?

[CLOUD](https://rapids.ai/cloud.html) -> [Deploy Docs](https://docs.rapids.ai/deployment/stable/)
DASK](https://rapids.ai/dask.html) -> [Dask Docs](https://docs.dask.org/en/stable/gpu.html)
[HPC](https://rapids.ai/hpc.html) -> [Deploy Docs](https://docs.rapids.ai/deployment/stable/)
[HPO](https://rapids.ai/hpo.html) -> [Deploy Docs](https://docs.rapids.ai/deployment/stable/)
[MERLIN](https://rapids.ai/merlin.html) -> [Nvidia Page](https://developer.nvidia.com/nvidia-merlin)
PLOTLY](https://rapids.ai/plotly.html) -> Build into new Viz gallery page
[XGBOOST](https://rapids.ai/xgboost.html) -> [XGBoost docs](https://xgboost.readthedocs.io/en/stable/gpu/index.html)

**Comment by bdice** [comment]
Yes, I believe these pages can all be deprecated / removed. Right now, I think the deploy docs are mostly pointing back to the rapids.ai site until content is migrated to the deployment docs. For example, the [deployment page for HPC](https://docs.rapids.ai/deployment/stable/hpc.html) says "Installation instructions for SLURM are hosted at the [RAPIDS HPC page](https://rapids.ai/hpc)." @mmccarty and @jacobtomlinson are working through this migration, to my knowledge.

The easier action items I see right now are Dask, Merlin, and XGBoost because those aren't waiting for content migration. If @aravenel agrees, I would be happy to remove those pages in favor of links to the respective pages mentioned above. The one risk we run is that by removing such content, we no longer own the messaging. The linked pages may not do as good of a job at emphasizing how RAPIDS _interacts_ with Dask/Merlin/XGBoost/etc. We want to have a clear message about our explicit integrations and emphasize that RAPIDS is more than just a friendly set of libraries that happen to have data analytics or GPU computations in common with the aforementioned partner libraries. (That said, it's certainly better to have less content if our messaging is unclear or outdated -- just wanted to point out that by linking out, we may sacrifice an opportunity to tell our side of the story if that matters.)

**Comment by mroeschke** [comment]
IMO the HPO seems like an application outcome of the deployment docs so I would say it belongs in the `cloud-ml-examples`

**Comment by jacobtomlinson** [comment]
I've opened #275 to redirect the cloud page.
